### PR TITLE
Modifications for parallel processing (scatter loops)

### PR DESCRIPTION
**Pull request by SpheMakh**
Goes with https://github.com/caracal-pipeline/stimela2/pull/10

### CODE REVIEW

File: scabha/cargo.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ParameterPolicies(object):
     # skip this parameter
     skip: bool = False
     # if True, implicit parameters will be skipped automatically
-    skip_implicits: bool = True
+    skip_implicits: bool = False
```
Comment (o-smirnov) [inline scabha/cargo.py]: @SpheMakh I think this needs to be None not False. The ``get_policy()`` logic relies on unset policies being None (i.e. if the parameter policy is None, use the cab policy). Setting it to True or False by default makes it impossible to simply specify one cab policy, as whatever the cab dev sets gets overridden by the parameter policy default.

File: scabha/cargo.py [hunk 2]
```diff
@@ -506,6 +504,7 @@ def stringify_argument(name, value, schema, option=None):
                 positional_first = get_policy(schema, 'positional_head') 
                 positional = get_policy(schema, 'positional') or positional_first
                 skip = get_policy(schema, 'skip') or (schema.implicit and get_policy(schema, 'skip_implicits'))
+                print(skip, schema.implicit, get_policy(schema, 'skip_implicits'))
```
Comment (o-smirnov) [inline scabha/cargo.py]: Remove debug print.

**Review by o-smirnov — CHANGES_REQUESTED** [review]
Don';t think skip_implicit is quite right!

File: scabha/cargo.py (hunk 2 above)
Comment (sphemakh) [reply scabha/cargo.py]: Done

File: scabha/cargo.py (hunk 1 above)
Comment (sphemakh) [reply scabha/cargo.py]: Agreed and changed

**Review by o-smirnov — APPROVED** [review]

**Pull request merged**